docs/services/slam/cartographer/_index.md

```diff
@@ -283,49 +201,13 @@ This example JSON configuration:
 
 For more information about the configuration attributes, see [Attributes](#attributes).
 
-1. Start a mapping session:
-
-   Navigate to the **CONTROL** tab on your machine's page and click on the dropdown menu matching the `name` of the service you created.
-   On the cartographer panel, you can start a mapping session.
-
-   When you start a mapping session, Cartographer uses the data captured from when you click **Start session** until you click **End session** to create the map.
-
-   Enter a name or use the suggested name for your new map and click **Start session**.
-   Wait for the slam session to finish starting up in the cloud, which **takes about 2 minutes**.
-
-   Make sure to either **manually refresh**, or **change the refresh frequency** to something other than `Manual`.
-
-   ![slam RC card start session](/services/slam/slam-RC-card-start-session.png)
-
-   While the slam session is starting, you will see a loading screen.
-
-   ![slam RC card wait for session to finish starting](/services/slam/slam-RC-card-wait-for-session-to-finish-starting.png)
-
-   Once the slam session has finished starting, your first pointcloud will appear.
-
-   ![slam RC card first pointcloud](/services/slam/slam-RC-card-first-pointcloud.png)
-
-   You can see that your cloud slam session is in progress from your **Location** page's **SLAM library** tab.
-
-   ![offline mapping maps computing table](/services/slam/offline-mapping-maps-computing-table.png)
-
-   When you are ready to end the slam session, return to your machine's **CONTROL** tab and click **End session**.
-   If you do not click **End session**, the slam session will automatically end after 45 minutes.
-
-   Once the session has ended, the map is saved to your **Location** page's **SLAM library** tab.
-
-   ![offline mapping available maps](/services/slam/offline-mapping-available-maps.png)
-
-   You can click `View map` to view the map in a dynamic pointcloud viewer.
-
-   ![slam library view map](/services/slam/slam-library-view-map.png)
+After configuring cartographer the machine should begin mapping automatically.  Navigate to the **CONTROL** tab on your machine's page and click on the dropdown menu matching the `name` of the service you created. See our [tips](#slam-mapping-best-practices) for making a good map! If you want to save your locally built map, you can use the **GetInternalState API** or use the local map uploading feature of the [cloudslam wrapper module](../cloudslam)
 
 ### Update an existing map
 
 To update an existing map with new pointcloud data from a new SLAM session, follow the instructions below.
-Updating an existing map uses an instance of the `cartographer` module running in the cloud, and _does not_ overwrite the existing map.
 
-1. Configure your `cartographer` SLAM service:
+1. Configure your `cartographer` SLAM service using a map from your location:
```

Comment:
I shouldn't have the number here, but idk what to put

Reply:
why shouldn't there be a number?

Reply:
just saw you made viewing its own step, which also works!